src/status_im/signals/core.cljs

```diff
@@ -77,4 +92,5 @@
       "local-notifications" (local-notifications/process cofx (js->clj event-js :keywordize-keys true))
       "community.found" (link.preview/cache-community-preview-data (js->clj event-js :keywordize-keys true))
       "status.updates.timedout" (visibility-status-updates/handle-visibility-status-updates cofx (js->clj event-js :keywordize-keys true))
+      "localPairing" (handle-local-pairing-signals event-str)
```

Comment:
Can we keep the convention and stick to event types that are lowercase and use dots to convey hierarchy? So something like `local-pairing` seems better.

Reply:
Since signal types are determined at status-go level, I will speak with @Samyoul  about this to make it a part of one of his existing PRs.
We can ignore this as it is for now I hope?